services/cards/src/actions/Cards.tsx

```diff
@@ -8,10 +8,51 @@ import {filtersCalculate} from './Filters';
 declare var require: any;
 const Tabletop = require('tabletop');
 
+interface ResultType {
+  cards: CardType[];
+  translations: TranslationsType;
+}
+
 export function downloadCards(): ((dispatch: Redux.Dispatch<any>) => void) {
   return (dispatch: Redux.Dispatch<any>) => {
     const store = getStore();
     dispatch(cardsLoading());
+
+    const keys = store.getState().filters.source.current.split(':')[1].split(',');
+
+    Promise.all(keys.map(downloadAndProcessSpreadsheet))
+      .then((results: any[]) => {
```

Comment:
This is a rather complex type to leave as `any`. Could you make this an interface?